src/containers/Booklist.js

```diff
@@ -33,13 +35,14 @@ BookList.propTypes = {
       category: PropTypes.string.isRequired,
     }),
   ).isRequired,
+  removeBook: PropTypes.func.isRequired,
 };
 const mapStateProps = (state) => ({
   books: state.books,
-
 });
 const mapDispatchToProps = (dispatch) => ({
   create: (book) => dispatch(CreateBook(book)),
-  delete: (book) => dispatch(RemoveBook(book)),
+  removeBook: (book) => dispatch(RemoveBook(book)),
+  // delete: (book) => dispatch(RemoveBook(book)),
 });
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))